assets/maps/prefabs/prefab_drug_den.dmm

```diff
@@ -0,0 +1,1885 @@
+/area/prefab/drug_den)
+"Pt" = (
+/obj/lattice{
+	dir = 1;
+	icon_state = "lattice-dir"
+	},
+/obj/item/clothing/under/rank/janitor{
+	pixel_x = 4;
+	pixel_y = -7
+	},
+/turf/space,
+/area/noGenerate)
+"Qa" = (
+/obj/item/reagent_containers/food/snacks/donkpocket{
+	pixel_x = -3;
+	pixel_y = 8
+	},
+/obj/item/reagent_containers/food/snacks/donkpocket{
+	pixel_x = 1;
+	pixel_y = 5
+	},
+/obj/decal/cleanable/vomit{
+	pixel_y = 9
+	},
+/turf/simulated/floor/carpet/arcade/half,
+/area/prefab/drug_den)
+"QJ" = (
+/obj/table/wood/auto,
+/obj/item/device/light/lava_lamp{
+	pixel_x = -6;
+	pixel_y = 16
+	},
+/obj/item/clothing/mask/cigarette/dryjoint{
+	pixel_x = -4;
+	pixel_y = -3
+	},
+/obj/item/clothing/mask/cigarette/dryjoint{
+	pixel_x = -1
+	},
+/obj/item/matchbook{
+	pixel_x = 6;
+	pixel_y = 7
+	},
+/turf/simulated/floor/wood,
+/area/prefab/drug_den/party)
+"QX" = (
+/obj/decal/cleanable/vomit{
+	pixel_x = -1;
+	pixel_y = 5
+	},
+/turf/simulated/floor/carpet{
+	dir = 5;
+	icon_state = "fblue2"
+	},
+/area/prefab/drug_den/party)
+"Rb" = (
+/obj/lattice{
+	icon_state = "lattice-dir-b"
+	},
+/turf/space,
+/area/noGenerate)
+"Rf" = (
+/obj/machinery/light/incandescent/netural{
+	dir = 1
+	},
+/obj/decal/cleanable/dirt/jen,
+/turf/simulated/floor/specialroom/cafeteria,
+/area/prefab/drug_den)
+"Si" = (
+/obj/item/device/light/glowstick/green_on{
+	pixel_x = 3;
+	pixel_y = 1
+	},
+/obj/machinery/light/incandescent/blueish,
+/obj/decal/cleanable/dirt/jen,
+/turf/simulated/floor/carpet/arcade/half,
+/area/prefab/drug_den)
+"SG" = (
+/obj/cable{
+	d2 = 2;
+	icon_state = "0-2"
+	},
+/turf/simulated/floor/carpet/arcade/half,
+/area/prefab/drug_den)
+"Tb" = (
+/obj/cable{
+	icon_state = "1-2"
+	},
+/turf/simulated/floor/carpet/purple/fancy/edge/west,
+/area/prefab/drug_den/party)
+"Ti" = (
+/obj/critter/mouse/mad,
+/turf/simulated/floor/wood,
+/area/prefab/drug_den/party)
+"Tv" = (
+/obj/lattice{
+	icon_state = "lattice-dir-b"
+	},
+/obj/machinery/light/runway_light/delay5,
+/turf/space,
+/area/noGenerate)
+"TA" = (
+/obj/lattice{
+	dir = 1;
+	icon_state = "lattice-dir-b"
+	},
+/obj/machinery/light/runway_light/delay5,
+/turf/space,
+/area/noGenerate)
+"TN" = (
+/obj/cable{
+	icon_state = "1-2"
+	},
+/obj/disposalpipe/segment{
+	dir = 8;
+	icon_state = "pipe-c"
+	},
+/obj/machinery/light/incandescent/netural{
+	dir = 4
+	},
+/turf/simulated/floor/wood,
+/area/prefab/drug_den/party)
+"TY" = (
+/obj/table/reinforced/auto,
+/obj/item/reagent_containers/food/snacks/donkpocket{
+	pixel_x = -3
+	},
+/obj/item/reagent_containers/syringe/jenkem{
+	pixel_x = -2;
+	pixel_y = 12
+	},
+/turf/simulated/floor/white/checker2{
+	dir = 1
+	},
+/area/prefab/drug_den)
+"Ur" = (
+/obj/cable{
+	icon_state = "4-8"
+	},
+/obj/disposalpipe/segment{
+	dir = 4
+	},
+/turf/simulated/floor/specialroom/cafeteria,
+/area/prefab/drug_den)
+"Uv" = (
+/turf/simulated/wall/auto/jen,
+/area/prefab/drug_den)
+"UD" = (
+/obj/table/auto,
+/obj/machinery/phone{
+	location = null
```

Comment:
Delete this, with the lates changes I made it will no longer compile (since the location var was unused anyway).
Also add either `phone_id = "whatever"` or `name = "whatever"` if you don't want the phone to be listed as "Space".

Reply:
I think the auto transformation of the file borked this.
I pushed the change manually; should work now. :)